### DISCUSSION TITLE
A question about building a milvus cluster in the ARM architecture

### DISCUSSION THREAD

**Discussion by XiaominZou** [milvus-io/milvus#28286] [Q&A and General discussion] [answered]
We  install milvus cluster with helm in the the ARM architecture. But we found the image  (milvusdb/milvus-config-tool:v0.1.1) only support  the x86 architecture. Does milvus cluster support the ARM architecture? How can I pull a milvus-config-tool for ARM architecture

**Comment by xiaofan-luan** [comment]
milvus-config-tool: should be purely go and it should be compiled?
Currently we only have milvus arm image and all the other image I thought there are no prebuilt one

**Reply by XiaominZou** [reply]
The x86 container cannot run in the ARM architecture, it will make the following error: Error: Start container error: runtime error: config:tools/lxc_start.c:main:410 starting container process caused "exec: "/cp": No such file or directory."
![image](https://github.com/milvus-io/milvus/assets/37055647/9c5d4a39-fa00-4ab4-b491-7b64e9f66ea3)

**Reply by xiaofan-luan** [reply]
@LoveEachDay 
Seems that we are trying to deploy x86 image for arm?

Do we need to compile all the tool image for arm also?

**Reply by LoveEachDay** [reply]
@xiaofan-luan We have hardcoded the `GOARCH` to `amd64` for `milvus-config-tool` image. Since milvus supports to run on `arm64`, we'd also provide all the tool image for arm.

**Reply by XiaominZou** [reply]
Where can I download milvus-config-tool image for ARM?

**Reply by LoveEachDay** [reply, marked as answer]
@XiaominZou Today we released the new version of `milvus-config-tool` for amd64 and arm64. 
Checkout: `milvusdb/milvus-config-tool:v0.1.2`.

**Reply by XiaominZou** [reply]
Appreciate it！
